src/jit/lir.h

```diff
@@ -262,6 +262,10 @@ class LIR final
         void InsertBefore(GenTree* insertionPoint, GenTree* node);
         void InsertAfter(GenTree* insertionPoint, GenTree* node);
 
+        void InsertBefore(GenTree* insertionPoint, GenTree* node1, GenTree* node2);
```

Comment:
Maybe we could use std::initializer_list<GenTree*> instead of creating so many overloads. Though it might be a bit slower than the overload version...